src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -231,7 +231,7 @@ private function extractFromAccessor($class, $property)
      */
     private function extractFromReflectionType(\ReflectionType $reflectionType)
     {
-        $phpTypeOrClass = (string) $reflectionType;
+        $phpTypeOrClass = method_exists($reflectionType, 'getName') ? $reflectionType->getName() : (string) $reflectionType;
```

Comment:
Because `__toString` has changed in PHP 7.1 and will change in the future (it is not properly part of the BC promise). As advised in the PHP 7.1 UPGRADING file, we need to use `ReflectionNamedType::getName` going forward.